src/extension/features/budget/link-to-inflows/index.js

```diff
@@ -0,0 +1,58 @@
+import { Feature } from 'toolkit/extension/features/feature';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import {
+  getSelectedMonth,
+  isCurrentRouteBudgetPage,
+  transitionTo,
+} from 'toolkit/extension/utils/ynab';
+
+/**
+ * Adds a click handler to the "TOTAL INFLOW" inspector area.
+ */
+export class LinkToInflows extends Feature {
+  shouldInvoke() {
+    return isCurrentRouteBudgetPage();
+  }
+
+  onRouteChanged() {
+    if (this.shouldInvoke()) {
+      this.invoke();
+    }
+  }
+
+  /**
+   * Add the wrapper element to the TOTAL INFLOWS inspector area.
+   */
+  invoke() {
+    $('.budget-inspector')
+      .find(`h3:contains("${Ember.I18n.t('inspector.totalIncome')}")`)
```

Comment:
funny story -- ynab doesn't actually use i18n (unless that's changed after a long time)

you'll want to use the `l10n` helper from `toolkit/extension/utils/toolkit` to do this. keys should be the same, you just need to provide a default

```
import { l10n } from 'toolkit/extension/utils/toolkit`;

....

l10n('inspector.totalIncome', 'TOTAL INFLOWS');
```

Reply:
Ah ok, good to know! 👍